widgetssdk/src/main/java/com/glia/widgets/fcm/GliaPushMessage.kt

```diff
@@ -0,0 +1,74 @@
+package com.glia.widgets.fcm
+
+/**
+ * Push object sent by the Glia client library
+ */
+interface GliaPushMessage {
+    /**
+     * Defines the possible types of the Glia push notifications
+     */
+    enum class PushType {
```

Comment:
It might be good to add tests for enums similar to [AuthenticationManagerTest.kt](https://github.com/salemove/android-sdk-widgets/pull/1259/files#diff-6b5ccc4cd0f9e87b9a4dcfe2b411de55371729fb24427f9e8280db62347d8858). To make sure they match the Core SDK enums.